js/script.js

```diff
@@ -0,0 +1,1286 @@
+
+const a = ['a', 'b', 'c'];
+const b = [1, 2, 3];
+
+const c = a.concat(b);
+console.log(c);
+
+
+//Дан массив ['a', 'b', 'c']. Добавьте ему в конец элементы 1, 2, 3.
+
+const a = ['a', 'b', 'c'];
+a.push(1, 2, 3);
+
+console.log(a);
+
+
+//Дан массив [1, 2, 3, 4, 5]. С помощью метода splice преобразуйте массив в [1, 4, 5].
+
+const a = [1, 2, 3, 4, 5];
+a.splice(1, 2);
+
+console.log(a);
+
+
+//Дан массив [1, 2, 3, 4, 5]. С помощью метода splice запишите в новый массив элементы [2, 3, 4].
+
+const a = [1, 2, 3, 4, 5];
+a.splice(5, 0, 2, 3, 4);
+
+console.log(a);
+
+
+//Дан массив [1, 2, 3, 4, 5]. С помощью метода splice сделайте из него массив [1, 2, 3, 'a', 'b', 'c', 4, 5].
+
+const a = [1, 2, 3, 4, 5];
+a.splice(3, 0, 'a', 'b', 'c');
+
+console.log(a);
+
+
+// Дан массив [1, 2, 3, 4, 5]. С помощью метода splice сделайте из него массив [1, 'a', 'b', 2, 3, 4, 'c', 5, 'e'].
+
+const a = [1, 2, 3, 4, 5];
+a.splice(1, 0, 'a', 'b');
+a.splice(6, 0, 'c');
+a.splice(8, 0, 'e');
+
+console.log(a);
+
+
+//Дан массив [3, 4, 1, 2, 7]. Отсортируйте его.
+
+const a = [3, 4, 1, 2, 7];
+a.sort();
+
+console.log(a);
+
+
+//Дан объект {js:'test', jq: 'hello', css: 'world'}. Получите массив его ключей.
+
+const a = {js:'test', jq: 'hello', css: 'world'};
+
+for (let key in a){
+  console.log(key);
+}
+
+
+//Дана строка, например, '123456'. Переверните эту строку (сделайте из нее '654321') не используя цикл. 
+
+const str = '123456';
+const a = str.split('').reverse().join('');
+
+console.log(a);
+
+
+//Дана строка. Сделайте заглавным первый символ этой строки не используя цикл. Найдите два решения
+
+const str = 'hello';
+const a = str[0].toUpperCase() + str.substring(1);
+
+console.log(a);
+
+
+//Проверьте, что строка начинается на http://.
+
+const str = 'http://do';
+
+const result = str.startsWith('http://');
+
+console.log(result);
+
+
+//Проверьте, что строка заканчивается на .html
+
+const str = 'do.html';
+
+const result = str.endsWith('.html');
+
+console.log(result);
+
+
+//Дан массив с числами. Проверьте, что в этом массиве есть число 5. Если есть - выведите 'да', а если нет - выведите 'нет'.
+
+const a = [1, 2, 3, 4, 5];
+let result = '';
+
+for (let i = 0; i < a.length; i++) {
+  if (a[i] === 5){
+    result = 'yes';
+  } else {
+    result = 'no';
+  }
+}
+
+console.log(result);
+
+
+//Дано число, например 31. Проверьте, что это число не делится ни на одно другое число кроме себя самого и единицы. То есть в нашем случае нужно проверить, что число 31 не делится на все числа от 2 до 30. Если число не делится - выведите 'false', а если делится - выведите 'true'.
+
+function checkNum (a) {
+  if ((a % 1 === a) && (a % 31 === 1)) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+console.log(checkNum(31));
+
+
+//Дан массив с числами. Проверьте, есть ли в нем два одинаковых числа подряд. Если есть - выведите 'да', а если нет - выведите 'нет'
+
+const arr =[1, 1, 2];
+
+function checkNum () {
```

Comment:
```suggestion
function checkNum() {
```
тут тоже никаких пробелов